apps/videos/urls.py

```
@@ -1,8 +1,7 @@
 # apps/videos/urls.py
-from django.urls import path
-from .views import UserVideoListView, VideoDeleteView
+from .views import UserVideoListView, VideoDeleteView, VideoDetailView
```

Ruff (0.12.2)

`.views.VideoDeleteView` imported but unused

Remove unused import: `.views.VideoDeleteView`

(F401)